social_core/tests/backends/open_id_connect.py

```diff
@@ -121,12 +122,12 @@ def get_id_token(
     def prepare_access_token_body(
         self,
         client_key=None,
-        tamper_message=False,
```

Comment:
Why was the argument removed when the code using it is still there? I believe it is better to list all arguments explicitly than parse them from kwargs.

Reply:
Done as NOQA.